Modules and packages/Executing modules as scripts/task.md

```diff
@@ -4,11 +4,10 @@ A Python module is a file containing executable statements as well as function o
 These statements are executed the first time the module name is encountered in an `import` statement.
 The file name is the module name with the suffix .py appended. Within a module, the 
 module’s name (as a string) is available as the value of the global variable `__name__`.
-A module can contain executable statements as well as function or class definitions. 
-These statements are executed the first time the module name is encountered in an `import` statement.
+
```

Comment:
I don't think we should delete this =)

Reply:
The problem is, these 2 sentences occur twice within the paragraph )